curvefs/src/client/mds_client.cpp

```diff
@@ -17,37 +17,226 @@
 
 /*
  * Project: curve
- * Created Date: Thur May 27 2021
- * Author: xuchaojie
+ * Created Date: Thur Jun 15 2021
+ * Author: lixiaocui
  */
 
 #include "curvefs/src/client/mds_client.h"
 
 namespace curvefs {
 namespace client {
 
-CURVEFS_ERROR MdsClientImpl::Init(const MdsOption& mdsOpt) {
-    return CURVEFS_ERROR::OK; }
+CURVEFS_ERROR MdsClientImpl::MDSRPCExcutor::DoRPCTask(RPCFunc task) {
+    brpc::Channel channel;
+    std::string mdsaddr = opt_.mdsaddr;
+    int ret = channel.Init(mdsaddr.c_str(), nullptr);
+    if (ret != 0) {
+        LOG(WARNING) << "Init channel failed, addr = " << mdsaddr;
+        return static_cast<CURVEFS_ERROR>(-EHOSTDOWN);
```

Comment:
-EHOSTDOWN is not include in enum CURVEFS_ERROR

Reply:
done